web/satellite/src/components/project/NewProjectPopup.vue

```diff
@@ -104,9 +109,17 @@
 
             this.fetchProjectMembers();
 
-            this.checkIfsFirstProject();
+            this.notifySuccess('Project created successfully!');
```

Comment:
Will just dispatching action here be more efficient since we are using it just once?